src/audiences/ui/list.js

```diff
@@ -103,13 +103,19 @@ class List extends Component {
 	 */
 	onSearch = event => {
 		const value = event.target.value;
+		const canCreate = wp.data.select( 'core' ).canUser( 'create', 'audiences' );
```

Comment:
I wasn't sure if there was a better/easier way to get `wp.data.select` but just using `select` here threw an error as it was not defined.

Reply:
Use `this.props.canCreate()` here